app/src/main/java/de/westnordost/streetcomplete/quests/bridge_structure/AddBridgeStructure.java

```diff
@@ -0,0 +1,53 @@
+package de.westnordost.streetcomplete.quests.bridge_structure;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+
+public class AddBridgeStructure extends SimpleOverpassQuestType{
+
+
+    @Inject
+    public AddBridgeStructure(OverpassMapDataDao overpassServer) {
+        super(overpassServer);
+    }
+
+    @Override
+    public AbstractQuestAnswerFragment createForm() {
+        return new AddBridgeStructureForm();
+    }
+
+    @Override
+    public int getIcon() {
+        return R.drawable.ic_quest_bridge;
+    }
+
+    @Override
+    public void applyAnswerTo(Bundle answer, StringMapChangesBuilder changes) {
+        changes.add("bridge:structure", answer.getString(AddBridgeStructureForm.STRUCTURE));
+    }
+
+    @Override
+    public String getCommitMessage() {
+        return "Add bridge structure";
+    }
+
+    @Override
+    public int getTitle(Map<String, String> tags) {
+        return R.string.quest_bridge_structure_title;
+    }
+    @Override
+    protected String getTagFilters() {
+        return "ways with bridge or man_made=bridge and !bridge:structure";
```

Comment:
This will ask the information twice for bridges mapped like this:

http://www.openstreetmap.org/#map=18/16.77657/96.18371
(road with `bridge=yes` + area with `man_made=bridge`)

As far as I know, the above method is the preferred (because most detailed) method. I propose to only ask the bridge structure on `man_made=bridge`.